source/dofs/dof_tools_constraints.cc

```diff
@@ -2517,18 +2529,21 @@ namespace DoFTools
 
     make_periodicity_constraints<DoFHandlerType>(matched_faces,
                                                  constraints,
-                                                 component_mask);
+                                                 component_mask,
+                                                 std::vector<unsigned int>(),
+                                                 periodicity_factor);
```

Comment:
Here and below. I need to pass the parameter `periodicity_factor` to `make_periodicity_constraints()`. The fourth parameter takes the default value. In order to pass `periodicity_factor` to `make_periodicity_constraints()` I added the default value to the fourth parameter'